src/Lykke.Logs/Loggers/LykkeSanitizing/SanitizingException.cs

```diff
@@ -7,11 +7,16 @@ internal sealed class SanitizingException : Exception
     {
         private readonly Exception _exception;
         private readonly Func<string, string> _sanitizer;
+        
+        private const string AsyncStackTraceExceptionData = "AsyncFriendlyStackTrace";
 
         public SanitizingException(Exception exception, Func<string, string> sanitizer)
         {
             _exception = exception ?? throw new ArgumentNullException(nameof(exception));
             _sanitizer = sanitizer ?? throw new ArgumentNullException(nameof(sanitizer));
+            
+            // add stack trace to be retrieved by AsyncFriendlyStackTrace later
+            this.Data.Add(AsyncStackTraceExceptionData, exception.StackTrace);
```

Comment:
Let's put new StackTrace(exception, true).ToAsyncString() instead of exception.StackTrace